sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceActionCancelEventData.java

```diff
@@ -59,13 +60,13 @@
      * The requested authorization for the operation.
      */
     @JsonProperty(value = "authorization")
-    private String authorization;
+    private ResourceAuthorization authorization;
```

Comment:
deprecate `authorization`  string getter and setter. and introduce a new getter and setter. ` [get/set]ResourceAuthorization`